gleam.toml

```diff
@@ -11,13 +11,13 @@ gleam = ">= 0.32.0"
 # links = [{ title = "Website", href = "https://gleam.run" }]
 
 [dependencies]
-gleam_stdlib = "~> 0.31"
-gleam_http = "~> 3.5"
-gleam_erlang = "~> 0.22"
-gleam_crypto = "~> 1.0"
-mist = "~> 1.2"
-sqlight = "~> 0.8"
-wisp = "~> 0.5"
+gleam_stdlib = "~> 0.38"
+gleam_http = "~> 3.6"
+gleam_erlang = "~> 0.25"
+gleam_crypto = "~> 1.3"
+sqlight = "~> 0.9"
+mist = ">= 1.2.0"
+wisp = ">= 1.0.0"
```

Comment:
Typically one does not edit this file to upgrade dependencies unless the desired version is outside the specified range, you can run `gleam update` instead.

Seeing as they are edited, please convert all the requirements from the `~>` syntax and specify an upper bound of the next major version for all of them. Thank you.

Reply:
Thanks. I have ever little experience with gleam. I did try gleam update but I was having issues resolving a set of dependencies that would compile together. I did some digging around in hex and found a set that worked for me. 

I will try again little later to get it working by resetting the file and using gleam update, or otherwise applying your recommendations above!

Thanks again and thanks for the language as well.

Reply:
The project is locked to working versions, so if you revert all your changes and run `gleam run` it will work with current Gleam.

You only need to make changes if you decide that you want to upgrade the dependencies.